### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by peterebden**
Also a few bugfixes:
 - handle multiline strings properly for built-in functions
 - die if URL is not passed to `plz follow`
 - Remove RunArgs config entry which isn't honoured any longer.

### CODE REVIEW

File: docs/config.html [hunk 1]
```diff
@@ -8,16 +8,17 @@ <h1>Please config file reference</h1>
...
+        <li><code>/etc/plzconfig</code></li>
+        <li><code>~/.please/plzconfig</code></li>
+        <li><code>.plzconfig</code></li>
+        <li><code>.plzconfig_&lt;arch&gt;</code> (e.g. .plzconfig_linux_amd64)</li>
+        <li><code>.plzconfig.&lt;profile&gt;</code> (if the <code>--profile</code> flag is passed)</li>
+        <li><code>.plzconfig.local</code></li>
       </ul>
-      One would normally check in .plzconfig (and arch-specific equivalents if needed).<br/>
+      One would normally check in <code>.plzconfig</code> (and arch-specific equivalents if needed).<br/>
       /etc/plzconfig is there to facilitate per-machine config in cases where the repo is often
```
Comment (agenticarus) [inline docs/config.html]: &lt;code&gt; here too

File: docs/config.html [hunk 2]
```diff
@@ -313,6 +358,9 @@ <h3>[Python]</h3>
         specified on the rule itself. Defaults to <code>python</code> but you could of course set it to
         <code>pypy</code>.</li>
 
+      <li><b>TestRunner</b><br/>
+        The test runner used to discover & run Python tests; one of unittest, pytest or behave.</li>
```
Comment (agenticarus) [inline docs/config.html]: &amp;amp;

Comment (agenticarus) [reply docs/config.html]: &lt;code&gt;unittest&lt;/code&gt; etc.

**Comment by peterebden** [comment]
Thanks, fixed those. Also cleaned up the flags stuff a bit (turns out go-flags was just fine, we were short-circuiting the error...).

**Pull request merged**